src/components/MapOL.tsx

```
@@ -0,0 +1,106 @@
+import { useEffect, useRef } from "react";
+import Map from "ol/Map";
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)